src/EditorFeatures/Test/Extensions/IProjectionBufferFactoryServiceExtensionsTests.cs

```diff
@@ -47,7 +48,8 @@ line 2
         [Fact]
         public void TestCreateProjectionBuffer()
         {
-            var exportProvider = TestExportProvider.ExportProviderWithCSharpAndVisualBasic;
+            var composition = EditorTestCompositions.Editor.AddParts(typeof(IProjectionBufferFactoryServiceExtensions));
```

Comment:
😕 shouldn't `IProjectionBufferFactoryServiceExtensions` be part of `EditorTestCompositions.Editor`?

Reply:
`IProjectionBufferFactoryServiceExtensions` is our interface, not from the Editor assemblies.

Reply:
I meant `EditorFeatures`. Whatever the name is for our editor features layer.

Reply:
It is part of EditorFeatures. We could use just that, but I was generally trying to use the lowest layer required. 
In this case the test seems to be testing editor behavior and only needs one extra type that defines a content type. The best would be to copy that content type definition to the test class and use that part as a "mock".

Reply:
Never mind, using EditorFeatures is good here.